tests/cases/notification-generator.html

```diff
@@ -205,6 +205,7 @@
               <option data-id="9" value="No icon">One action (no icon)</option>
               <option data-id="10" value="$$/resources/icons/11.png">One action (empty title)</option>
               <option data-id="11" value="Reply$$/resources/icons/11.png$$text">One action (type text)</option>
```

Comment:
Just noticed that when I added this action I gave it the same data-id field as an existing option, on line 201, since they're not in order I didn't spot it - oops!

shall I fix it up now & change the new ones to 12 & 13 instead of further confusing things?